libs/remix-debug/test/decoder/vmCall.ts

```diff
@@ -39,31 +40,28 @@ export function sendTx (vm, from, to, value, data, cb) {
 }
 
 function createVm (hardfork) {
-  // const stateManager = new StateManagerCommonStorageDump({})
-  // stateManager.checkpoint(() => {})
-  const vm = new EthJSVM({
-    activatePrecompiles: true,
-    hardfork
-  })
-  vm.blockchain.validate = false
+  const common = new Common({ chain: 'mainnet', hardfork })
```

Comment:
Not related to the stack underflow error, but you should use this common instance you create here also for all the tx and block instantiations, otherwise you'll run into inconsistencies since these are then operating with the rules of another hardfork.

Reply:
thanks , I'll check that